### ISSUE TITLE
Apiserver fails to generate certs on fedora

### ISSUE THREAD

**Issue by bprashanth** [kubernetes/kubernetes#10725]
Setting up a kubernetes cluster on a couple of gce-fedora machines (per https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/getting-started-guides/fedora/fedora_manual_config.md) I ran into:

```
$ sudo systemctl status kube-scheduler -l
c.host%21%3D&resourceVersion=29: dial tcp 10.240.255.44:8080: connection refused
Jul 05 01:23:25 fed-master kube-scheduler[2574]: E0705 01:23:25.474400    2574 reflector.go:162] Failed to watch *api.Pod: Get http://fed-master:8080/api/v1beta3/watch/pods?fieldSelector=spec.host%3D&resourceVersion=29: dial tcp 10.240.255.44:8080: connection refused
Jul 05 01:23:25 fed-master kube-scheduler[2574]: E0705 01:23:25.474413    2574 reflector.go:162] Failed to watch *api.Node: Get http://fed-master:8080/api/v1beta3/watch/nodes?fieldSelector=spec.unschedulable%3Dfalse&resourceVersion=29: dial tcp 10.240.255.44:8080: connection refused
```

Think it's because the apiserver fails to generate certs in /var/run/kubernetes. Creating the dir and chowning it as kube worked. This didn't happen on the node, so I suspect it's a kube vs root permissions issue.

We should either make the installation smarter or update the docs.

@eparis @wattsteve

**Comment by eparis** [comment]
Actually, it's likely because the apiserver isn't even listening on 8080 on anything other than 127.0.0.1....

**Comment by eparis** [comment]
I wonder if we should scrap the manual docs and just tell people to use ansible (The ansible in contrib/ansible/ not the ansible in the kubernetes-ansible repo, so that doc needs updated too)

**Comment by bprashanth** [comment]
The apiserver might've crashed when it hit the permissions error, unsure, but I was able to create pods etc after I chowned the directory and restarted services. 

Having a single canocial fedora guide makes sense. Ansible is an external dependency, but it should reduce the maintenance overhead. Optionally we could explain what ansible does behind the scenes, since the manual guide provides some debug insight.
